src/components/LoadingWithoutModal/index.js

```diff
@@ -7,7 +7,7 @@ const LoadingWithoutModal = (props) => {
   return (
     <View style={styles.container}>
       <View style={styles.content(!!props.text)}>
-        <ActivityIndicator size="large" color={COLORS.blue} />
+        <ActivityIndicator size="large" color={COLORS.holytosca} />
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The color constant has been changed from `COLORS.blue` to `COLORS.holytosca`. However, the import statement for colors was supposed to be updated from `COLORS` to `colors` as per the PR description. Please ensure that the correct color constant is being used.
```diff
- <ActivityIndicator size="large" color={COLORS.holytosca} />
+ <ActivityIndicator size="large" color={colors.holytosca} />
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->